api/src/pcapi/workers/push_notification_job.py

```diff
@@ -31,3 +36,24 @@ def send_offer_link_by_push_job(user_id: int, offer_id: int) -> None:
     offer = Offer.query.get(offer_id)
     notification_data = get_offer_notification_data(user_id, offer)
     send_transactional_notification(notification_data)
+
+
+@job(worker.default_queue)
+def send_unretrieved_bookings_from_offer_notification_job(booking_ids: list[int]) -> None:
+    """
+    Send a notification to each of the unexpired bookings.
+    Filter bookings that will expire too soon to avoid a useless
+    notification.
+    """
+    expires_at_min = datetime.utcnow() - timedelta(hours=1)
+    bookings = bookings_repository.get_unexpired_bookings(booking_ids, expires_at_min)
+
+    for booking in bookings:
+        notification_data = get_unretrieved_bookings_with_offers_notification_data(booking)
+
+        try:
+            send_transactional_notification_delayed(notification_data)
+        except TypeError:
+            logger.error(
```

Comment:
```suggestion
            logger.exception(
```

Ainsi, Sentry nous montrera un traceback complet, ce sera plus facile de trouver la cause du problème.